LetsBake/Timer/View/TimerViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  TimerViewController.swift
+//  LetsBake
+//
+//  Created by Daye on 2021/12/09.
+//
+
+import UIKit
+
+class TimerViewController: UIViewController {
+
+  // MARK: - Properties
+
+  var hour: Int = 0
+  var minute: Int = 0
+  var second: Int = 0
```

Comment:
```suggestion
private  var hour: Int = 0
private  var minute: Int = 0
private  var second: Int = 0
```
외부에서 참조하여 사용할 경우가 아니라면 이처럼 접근 지정자를 선언해주는게 좋을듯 싶어요!